packages/react-vis/src/animation.js

```diff
@@ -59,6 +59,7 @@ class Animation extends PureComponent {
     this._updateInterpolator(props);
   }
 
+  // eslint-disable-next-line react/no-deprecated
   componentWillUpdate(props) {
```

Comment:
What bout renaming it to `UNSAFE_ componentWillUpdate` instead ?

That way we don't need an `eslint-disable` and we are being transparent about the fact that we use unsafe methods that need to be fixed easily

Reply:
same comment for all the `// eslint-disable-next-line react/no-deprecated` that were added

Reply:
It depends on what the minimum version of react that we want to support, and what version the `UNSAFE_` prefix was introduced.
Since we want to use hooks the lowest we can go is `16.8` but I haven't looked to see if there is anything else that we'd like to have that is in a version later than that.

Reply:
Indeed you're right! Let's ignore the lint error for now then, good call

Reply:
It looks like the `UNSAFE_` prefixes were added in `react@16.3` so we should be safe to move over to the `UNSAFE_` variants.